launch/tier4_autoware_api_launch/launch/deprecated_api.launch.xml

```diff
@@ -1,4 +1,6 @@
 <launch>
+  <arg name="api_mode" default="0"/>
```

Comment:
Could you add description for this argument? (LIke this [line](https://github.com/autowarefoundation/autoware_launch/blob/a75b0536f244762821ad5077299834b75656b85d/autoware_launch/launch/components/tier4_perception_component.launch.xml#L49) in autoware_launch)